templates/lookup_identity.conf.erb

```diff
@@ -1,5 +1,5 @@
 
-<LocationMatch ^/users/(ext)?login/?$>
+<LocationMatch ^(/api)?/users/(ext)?login/?$>
```

Comment:
This would also match `/api/users/login` which doesn't exist in foreman. Is that an issue?